main.rb

```diff
@@ -0,0 +1,167 @@
+require_relative 'app'
+
+def book_list(app)
+  print "Book list:\n"
+  if app.books.length.positive?
+    app.list_books
+    print 'Press enter to continue'
+  else
+    print 'There are not any books added yet'
+  end
+  gets.chomp
+end
+
+def people_list(app)
+  if app.people_exist?
+    app.list_people
+    print "\nPress enter to continue\n"
+  else
+    print "There are not any people added yet\n"
+    print "Press enter to continue\n"
+  end
+  gets.chomp
+end
+
+def create_student(app, type)
+  print 'Age: '
+  age = gets.chomp
+  print 'Name: '
+  name = gets.chomp
+  age = age.to_i # Convert age from string to integer
+  app.create_person(type, age, name)
+  print 'Person created successfully(press enter)'
+  gets.chomp
+end
```

Comment:
- When creating a student, it is expected to ask the question `Has parent permission? [y/n]:` But your app skips that part. please, kindly implement this.